src/source/PeerConnection/PeerConnection.c

```diff
@@ -782,8 +782,9 @@ STATUS addTransceiver(PRtcPeerConnection pPeerConnection, PRtcMediaStreamTrack p
             CHK(FALSE, STATUS_NOT_IMPLEMENTED);
     }
 
+    //TODO: Add ssrc duplicate detection here not only relying on RAND()
     CHK_STATUS(createKvsRtpTransceiver(RTC_RTP_TRANSCEIVER_DIRECTION_SENDRECV, pKvsPeerConnection, (UINT16) RAND(),
-                                       pRtcMediaStreamTrack, NULL, pRtcMediaStreamTrack->codec, &pKvsRtpTransceiver));
+            (UINT16) RAND(), pRtcMediaStreamTrack, NULL, pRtcMediaStreamTrack->codec, &pKvsRtpTransceiver));
```

Comment:
Can we please have stack based UINT16 variables which get set calling RAND and those be passed into the function? It's not a good style to call a macro in the param